stable/spinnaker/values.yaml

```diff
@@ -109,27 +85,20 @@ minio:
   serviceType: ClusterIP
   accessKey: spinnakeradmin
   secretKey: spinnakeradmin
+  bucket: "spinnaker"
   nodeSelector: {}
 
 gcs:
   enabled: false
   project: my-project-name
+  bucket: "<GCS-BUCKET-NAME>"
   jsonKey: '<INSERT CLOUD STORAGE JSON HERE>'
 
-# Configuration for the Jenkins instance that is attached to the
-# igor component of Spinnaker. For more info about the Jenkins integration
-# with Spinnaker, visit:
-#   https://www.spinnaker.io/setup/ci/jenkins/
-jenkins:
-  enabled: true
-  Master:
```

Comment:
does this need to be capitalized for an external dependency?  otherwise it would be nice to lowercase it to remain consistent.

Reply:
This is indeed how the Jenkins charts naming works. That said, the dependency on Jenkins has been removed.